src/services/api.js

```diff
@@ -1,7 +1,12 @@
 export async function getCategories() {
-  // Começando !!
+  const endPoint = 'https://api.mercadolibre.com/sites/MLB/categories';
+  const response = await fetch(endPoint);
+  return  await response.json();
 }
 
-export async function getProductsFromCategoryAndQuery(/* categoryId, query */) {
-  // Implemente aqui! Quando o fizer, descomente os parâmetros que essa função recebe
+export async function getProductsFromCategoryAndQuery(categoryId, query) {
+  const endPoint = `https://api.mercadolibre.com/sites/MLB/search?category=${categoryId}&q=${query}`;
+  const response = await fetch(endPoint);
+  const object = await response.json();
+  return object;
```

Comment:
O que você acha da gente repetir o mesmo retorno da outra função?
```suggestion
  return await response.json();
```